Makefile

```diff
@@ -99,32 +88,33 @@ release-binaries: ## Build release binaries
 test: generate verify ## Run tests
 	bazel test --nosandbox_debug //pkg/... //cmd/... $(BAZEL_ARGS)
 
+.PHONY: install-dev
+	install bazel-bin/cmd/clusterawsadm/${BINARYPATHPATTERN}/clusterawsadm $(shell go env GOPATH)/bin/clusterawsadm
+	install bazel-bin/cmd/clusterctl/${BINARYPATHPATTERN}/clusterctl $(shell go env GOPATH)/bin/clusterctl
+
 verify:
 	./hack/verify_boilerplate.py
 
 .PHONY: copy-genmocks
 copy-genmocks: ## Copies generated mocks into the repository
 	cp -Rf bazel-genfiles/pkg/* pkg/
 
-BAZEL_DOCKER_ARGS_COMMON := --define=MANAGER_IMAGE_NAME=$(MANAGER_IMAGE_NAME) --define=MANAGER_IMAGE_TAG=$(MANAGER_IMAGE_TAG) $(BAZEL_ARGS)
-BAZEL_DOCKER_ARGS := --define=DOCKER_REPO=$(STABLE_DOCKER_REPO) $(BAZEL_DOCKER_ARGS_COMMON)
-BAZEL_DOCKER_ARGS_DEV := --define=DOCKER_REPO=$(DEV_DOCKER_REPO) $(BAZEL_DOCKER_ARGS_COMMON)
-
 .PHONY: docker-build
-docker-build: generate ## Build the production docker image
+docker-build: generate provider-components ## Build the production docker image
 	bazel run //cmd/manager:manager-image $(BAZEL_DOCKER_ARGS)
 
-.PHONY: docker-build-dev
-docker-build-dev: generate ## Build the development docker image
-	bazel run //cmd/manager:manager-image $(BAZEL_DOCKER_ARGS_DEV)
-
 .PHONY: docker-push
-docker-push: generate ## Push production docker image
+docker-push: generate provider-components ## Push production docker image
```

Comment:
Am I missing the provider-components target? I only see provider-components-dev added